MODULE.bazel

```diff
@@ -51,7 +51,7 @@ bazel_dep(
 )
 bazel_dep(
     name = "rules_jvm_external",
-    version = "5.3",
+    version = "6.0",
```

Comment:
Could you please update the `repositories.bzl` file too? The versions of `rules_jvm_external` needs to match both here and there.